README.md

```diff
@@ -274,26 +267,26 @@ To learn more about the Nightscout API, visit https://YOUR-SITE.com/api-docs.htm
   * `DEBUG_MINIFY` (`true`)  - Debug option, setting to `false` will disable bundle minification to help tracking down error and speed up development
   * `DE_NORMALIZE_DATES`(`true`) - The Nightscout REST API normalizes all entered dates to UTC zone. Some Nightscout clients have broken date deserialization logic and expect to received back dates in zoned formats. Setting this variable to `true` causes the REST API to serialize dates sent to Nightscout in zoned format back to zoned format when served to clients over REST.
 
-
 ### Predefined values for your browser settings (optional)
+
   * `TIME_FORMAT` (`12`)- possible values `12` or `24`
   * `NIGHT_MODE` (`off`) - possible values `on` or `off`
   * `SHOW_RAWBG` (`never`) - possible values `always`, `never` or `noise`
-  * `CUSTOM_TITLE` (`Nightscout`) - Usually name of T1
-  * `THEME` (`default`) - possible values `default`, `colors`, or `colorblindfriendly`
```

Comment:
according to source `default` is still the default, see https://github.com/nightscout/cgm-remote-monitor/blob/f81298fb287e4a93c4e05ef23bca5b2ac2bf6b78/views/index.html#L259

Reply:
Ah, yes. `colors` has been the _de facto_ default for quite a while, as it's pre-populated in the Heroku deployment template. Should update the index.html to reflect this.

Reply:
Changing defaults should not be done in a patch release. @sulkaharo: can you reflect your view on this?

Reply:
Was changed as the pre-populated default in the template back in early February after data gathering and consultation in the FB groups — vast majority of end users switch to `colors`, so made sense to default it.